app/assets/stylesheets/components/_index.scss

```diff
@@ -6,3 +6,4 @@
 @import "card";
 @import "grid-index";
 @import "tab";
+@import "show";
```

Comment:
Move this import from components to pages/_index.scss

Reply:
@remi-www + can you check the conflict indicated by Github please?
You just to clean the import file I guess